proto/pulumi/provider.proto

```diff
@@ -279,6 +283,9 @@ message ProviderHandshakeResponse {
     // True if the provider accepts and respects autonaming configuration that the engine provides on behalf of the
     // user. *Must* match the value returned in response to [](pulumirpc.ResourceProvider.Configure).
     bool supports_autonaming_configuration = 4;
+
+    // True if the provider creates view resources and needs an address of the resource status service.
+    bool has_views = 5;
```

Comment:
Do we need this? We're only starting the status service up once now, we may as well just always start it up and send it to all providers?

Reply:
I had the same question annd ended up keeping it because there is still a small amount of tracking overhead related to the `token` that we could avoid for providers that don’t have views.

Reply:
Removed it.